CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.2.1 (2024-04-27)
 
+* Added option to mint pulic notes in the faucet (#339).
```

Comment:
This should go into `v0.3.0` section (you may need to rebase from the latest `next`).